tests/test_summary.py

```diff
@@ -86,8 +86,9 @@ def test_df_column_keys():
     assert set(sumdf.columns) == {"FOPT", "FOPR"}
     assert set(sumdf.attrs["meta"].keys()) == {"FOPT", "FOPR"}
 
-    with pytest.raises(ValueError, match="No valid key"):
-        summary.df(EclFiles(REEK), column_keys=["BOGUS"])
+    sumdf = summary.df(EclFiles(REEK), column_keys=["BOGUS"])
+    assert sumdf.columns.empty
+    assert not sumdf.index.empty
```

Comment:
something like `assert all(sumdf_no_keys.index == sumdf.index)` would be even more precise

Reply:
Deleted the previous comment. Found a better solution